WalletWasabi.Tests/UnitTests/Stores/BlockFilterSqliteStorageTests.cs

```diff
@@ -0,0 +1,158 @@
+using NBitcoin;
+using System.IO;
+using WalletWasabi.Backend.Models;
+using WalletWasabi.Blockchain.BlockFilters;
+using WalletWasabi.Stores;
+using Xunit;
+
+namespace WalletWasabi.Tests.UnitTests.Stores;
+
+/// <summary>
+/// Tests for <see cref="BlockFilterSqliteStorage"/>.
+/// </summary>
+public class BlockFilterSqliteStorageTests
+{
+	[Fact]
+	public void TryAppend()
+	{
+		FilterModel startingFilter = StartingFilters.GetStartingFilter(Network.Main);
+
+		string storagePath = $"{nameof(UnitTests)}.{nameof(BlockFilterSqliteStorage)}.{nameof(TryAppend)}.sqlite";
+		File.Delete(storagePath);
```

Comment:
We can enter into an verbal aikido combat about this but I really really would prefer in-memory db for UTs.
 
```suggestion
		string storagePath = ":memory:";
```